### ISSUE TITLE
Minimum password length consistency in translation files

### ISSUE THREAD

**Issue by MattOnyx** [jhipster/generator-jhipster#10266]
##### **Overview of the issue**

The min length for passwords is 4 in components but is still 5 in several i18n files (checked fr, ja).

##### **Motivation for or Use Case**

The password length should be the same everywhere.

##### **Reproduce the error**

Generate project in French or Japanese.

##### **Related issues**

None.

##### **Suggest a Fix**

PR in progress. Changing numbers.

##### **JHipster Version(s)**

6.2.0

##### **JHipster Version(s)**

```
espaceclient@0.0.0 C:\project\menergies-espace-client
`-- generator-jhipster@6.2.0 

```

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**

<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "xxx",
      "nativeLanguage": "fr"
    },
    "jhipsterVersion": "6.2.0",
    "applicationType": "monolith",
    "baseName": xxx
    "packageName": "xxx",
    "packageFolder": "xxx",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "cacheProvider": "no",
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "mssql",
    "prodDatabaseType": "mssql",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "maven",
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "xxx",
    "useSass": true,
    "clientPackageManager": "npm",
    "clientFramework": "angularX",
    "clientTheme": "none",
    "clientThemeVariant": "",
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "entitySuffix": "",
    "dtoSuffix": "DTO",
    "otherModules": [],
    "enableTranslation": true,
    "nativeLanguage": "fr",
    "languages": ["fr"],
    "blueprints": []
  }
}

</pre>
</details>

##### **Environment and Tools**

java version "1.8.0_201"
Java(TM) SE Runtime Environment (build 1.8.0_201-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.201-b09, mixed mode)

git version 2.21.0.windows.1

node: v10.14.2

npm: 6.5.0

yeoman: 3.1.0

yarn: 1.13.0

Docker version 19.03.1, build 74b1e89

docker-compose version 1.24.1, build 4667896b

INFO! Congratulations, JHipster execution is complete!

-   [x] Checking this box is mandatory (this is just to show you read everything)

**Comment by pmverma** [comment]
`PR in progress. Changing numbers.
`
Thanks for reporting the issue and doing the PR.

**Comment by pmverma** [comment]
Closing as this has been fixed #10267
